cmd/tailscaled/tailscaled.go

```diff
@@ -96,6 +97,34 @@ var subCommands = map[string]*func([]string) error{
 	"debug":                   &debugModeFunc,
 }
 
+func tryWindowsAppDataMigration() {
+	if runtime.GOOS != "windows" {
+		return
+	}
+
+	localAppData := filepath.Join(os.Getenv("LocalAppData"), "Tailscale")
+	_, err := os.Stat(localAppData)
+	if os.IsNotExist(err) {
+		return
+	}
+	if err != nil {
+		log.Printf("tryWindowsAppDataMigration failed; no AppDataLocal: %v", err)
+		return
+	}
+
+	programData := filepath.Join(os.Getenv("ProgramData"), "Tailscale")
+	_, err = os.Stat(programData)
+	if err == nil || os.IsNotExist(err) {
```

Comment:
if it already exists (err == nil), don't we want to bail and not try to move a directory atop (in?) a directory?

Reply:
Oh, yeah that is an important one. Logged a message if both ProgramData and AppData\Local exist, but since there is a legitimate case where that can happen, don't do anything else.